src/config/config_def.c

```diff
@@ -288,7 +288,8 @@ static const WT_CONFIG_CHECK confchk_WT_SESSION_commit_transaction[] = {
   {"sync", "string", NULL, "choices=[\"off\",\"on\"]", NULL, 0}, {NULL, NULL, NULL, NULL, NULL, 0}};
 
 static const WT_CONFIG_CHECK confchk_WT_SESSION_compact[] = {
-  {"timeout", "int", NULL, NULL, NULL, 0}, {NULL, NULL, NULL, NULL, NULL, 0}};
+  {"free_space_target", "int", NULL, "min=0", NULL, 0}, {"timeout", "int", NULL, NULL, NULL, 0},
```

Comment:
What happens if we set `free_space_target=0` does compact continue indefinitely?

Reply:
This means we would always satisfy the criteria for compaction and we will always try to compact. However, if no progress is made, compact quits.

Reply:
Okay, thanks